crates/ruff_python_ast/src/nodes.rs

```diff
@@ -3386,6 +3386,24 @@ pub enum TypeParam {
     TypeVarTuple(TypeParamTypeVarTuple),
 }
 
+impl TypeParam {
+    pub fn name(&self) -> &Identifier {
+        match self {
+            Self::TypeVar(x) => &x.name,
+            Self::ParamSpec(x) => &x.name,
+            Self::TypeVarTuple(x) => &x.name,
+        }
+    }
+
+    pub fn default(&self) -> Option<&Expr> {
+        match self {
+            Self::TypeVar(x) => x.default.as_deref(),
+            Self::ParamSpec(x) => x.default.as_deref(),
+            Self::TypeVarTuple(x) => x.default.as_deref(),
+        }
+    }
```

Comment:
hmm... I see that this could be useful, but the name could be confusing, since most `default()` methods come from the [`Default` trait](https://doc.rust-lang.org/std/default/trait.Default.html). I don't necessarily have a better suggestion, though...!

Reply:
I think this is fine, considering that it is aligned with the field name